cdisc_rules_engine/models/dataset_types.py

```diff
@@ -4,5 +4,6 @@
 class DatasetTypes(BaseEnum):
     CONTENTS = "contents"
     METADATA = "metadata"
+    RAW_METADATA = "raw_metadata"
     JSON_METADATA = "json_metadata"
```

Comment:
Can we delete this dataset_type as well?

Reply:
Sure, thanks for noting it